lsp/nls/src/requests/hover.rs

```diff
@@ -57,10 +58,18 @@ pub fn handle(
                     language: "nickel".into(),
                     value: ty.to_string(),
                 }),
-                MarkedString::LanguageString(LanguageString {
-                    language: "plain".into(),
-                    value: meta.join("\n"),
-                }),
+                MarkedString::String(
+                    meta.iter()
+                        .map(|s| {
+                            s.lines()
+                                .map(|s| if s.is_empty() { " " } else { s })
```

Comment:
Fixes a strange regression with the `unindent` crate that causes strings like this to work:
![image](https://github.com/tweag/nickel/assets/89555032/cd26f89a-c591-44d2-8a57-2692ea95c2b8)

But not this:
![image](https://github.com/tweag/nickel/assets/89555032/248e491c-c143-4802-9477-f07681ee0956)